torch/csrc/distributed/rpc/rpc_agent.h

```diff
@@ -112,6 +112,9 @@ class TORCH_API RpcAgent {
   // start accepting requests
   virtual void start() {}
 
+  // stop accepting requests
+  virtual void localShutdown() {}
```

Comment:
Call this `shutdown` and also expand the comment: "Stop accepting requests and shutdown the RPC framework as soon as possible by terminating all RPC threads"